### ISSUE TITLE
checkra1n isn't working on iOS 12.1.4 (iPhone 6s)

### ISSUE THREAD

**Issue by AngryKiller** [checkra1n/BugTracker#677]
**Tell us about your setup:**
1. iPhone 6s
2. 12.1.4
3. Tried with GUI and CLI
4. macOS 10.14.6 on a 2013 MBP
5. Tried with an Apple USB-A to Lightning cable, and a USB-A to 30-pin cable with Apple 30-pin to Lightning adapter

**What are the steps to reproduce the issue?**
1. Plug in the iPhone
2. Launch checkra1n
3. If you're using the GUI, put it in recovery first to bypass the firmware check, and if you're using the CLI just put it in DFU
4. Follow checkra1n's instructions
...

**What do you expect, and what is happening instead?**
Well, I didn't really expected it to work since it isn't officially supported, but it would be nice if it was
Instead, this is what I get:

![IMG_0930](https://user-images.githubusercontent.com/16365421/71450487-67816400-2763-11ea-95b9-20b4109d3889.JPG)

**Does the issue also occur if you enable Safe Mode?**
Yes

**Any other info, error logs, screenshots, ...?**

Already shown in the "what's happening" section

**Comment by BITBOARDDEV** [comment]
Same exact issue on iPhone SE iOS 12.1 with both CLI and GUI on a macOS KVM

**Comment by quilenle** [comment]
@AngryKiller @MiekMcSandvich 
Did you read this line before you download the tool?
![image](https://user-images.githubusercontent.com/50391899/71498830-8f221a80-2890-11ea-9ed4-8b5155bc03bb.png)

The team has written down the specific version that is supported. So why you ask the question like that?

You can close the issue now.

Best regards.

**Comment by shtuseriff** [comment]
Because this has been done successfully before. So, even though it's not officially supported, the fact that it works on some phones but not on others is strange.
Anyway, same issue here.

**Comment by AngryKiller** [comment]
> @AngryKiller @MiekMcSandvich
> Did you read this line before you download the tool?
> ![image](https://user-images.githubusercontent.com/50391899/71498830-8f221a80-2890-11ea-9ed4-8b5155bc03bb.png)
> 
> The team has written down the specific version that is supported. So why you ask the question like that?
> 
> You can close the issue now.
> 
> Best regards.

I've seen that, which is why I said that I didn't expected it to work, this is a request to add official support for those firmwares if this is possible.

**Comment by wbinder1** [comment]
I too would appreciate this, I have an iPhone 7 Plus on iOS 12.1

**Comment by filipef101** [comment]
Yeah, this would be really handy as I don't want to upgrade to 13
